src/resources/scripts/toCoco.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+import numpy
+from shapely.geometry import Point
+from shapely.affinity import scale,rotate
+import warnings
+import sys
+import math
+import json
+import ast
+
+# target_file = sys.argv[2]
+# csv = sys.argv[3]
+
+# the path to the CSV file
+paths = sys.argv[3:]
+# json (output) path #ignore for now
+save_json_path = sys.argv[2]
```

Comment:
Remove if this is really ignored? Instead, maybe add a comment to the PHP code to make the behavior of the script clear (see my other comment).